test/Base_Tests/src/Runtime/Stack_Traces_Spec.enso

```diff
@@ -144,6 +144,17 @@ add_specs suite_builder = suite_builder.group "Stack traces" group_builder->
             (deep_stack_trace.length > 5) . should_be_true
             deep_stack_trace.take 5 . map .name . should_equal names
 
+    group_builder.specify "source location to Text" <|
```

Comment:
Worth adding a comment: "This test must be the last, because it uses an absolute location".

Reply:
- It doesn't have to be the least test
- Yes, the absolute position requires updates when something above changes
- just like in fd4686f